pkg/certifier/clearlydefined/clearlydefined.go

```diff
@@ -153,7 +156,6 @@ func (c *cdCertifier) CertifyComponent(ctx context.Context, rootComponent interf
 
 	packMap := map[string][]*root_package.PackageNode{}
 	for _, node := range packageNodes {
-		// skip any purls that are generated by GUAC as they will not be found in clearly defined
```

Comment:
Why remove this comment?

Reply:
I removed the comments while tweaking the code, I forgot to add them back, thanks!